network-api/networkapi/templates/fragments/buyersguide/primary_nav.html

```diff
@@ -4,6 +4,7 @@
 
 
 {% block nav_logo %}
+{% get_bg_home_page as home_page %}
```

Comment:
Having to define this tag within each block seems a little weird to me, however, according to the django docs this is by design:
https://docs.djangoproject.com/en/4.1/howto/custom-template-tags/
<img width="849" alt="Screen Shot 2022-08-22 at 2 13 24 PM" src="https://user-images.githubusercontent.com/18314510/186020205-35589e3f-0b51-4c0d-8bd4-577f974afa8d.png">